ONSdigital/response-operations-ui
# Search respondents ui

# Motivation and Context
UI changes to allow searching of respondents by first name, last name, and email, rather than just by email.

# What has changed
* Views changed to include first name and last name fields,
* Search form separation for respondents and reporting units
* Added mock party controller to allow testing locally without party connection
* Added mock data for mock controller
* Added new results view page, with pagination
* Styling
* Created utils helper for respondent section of site
* Unit tests
* Added new UI tests to avoid adding non-functional tests to acceptance tests repo
* Added some tests for previously untested areas of code
* Updated to pattern library 2.0.0

# How to test?
* Run locally with `PARTY_FAKE_DATA` variable set to 1
* Confirm you can search respondents (you can't click through to a dummy respondent)
* Acceptance tests not yet addressed

# Links
https://trello.com/c/Mi1Zpw1l/707-us215-front-end-search-for-a-respondents-account-name-part-of-email-address-8

**When approved, don't merge - I need to merge acc tests at the same time**
